.github/workflows/remove-wheels.yml

```diff
@@ -0,0 +1,75 @@
+name: Remove old wheels
+
+on:
+  # Run daily at 1:23 UTC
+  schedule:
+  - cron:  '23 1 * * *'
+  workflow_dispatch:
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+# Needed for micromamba pickup
+defaults:
+  run:
+    shell: bash -l {0}
+
+env:
+  N_LATEST_UPLOADS: 5
+  ANACONDA_USER: "scientific-python-nightly-wheels"
+
+jobs:
+  remove:
+
+    runs-on: ubuntu-latest
+    environment:
+      name: remove-old-wheels
+    strategy:
+      matrix:
+        package-name: [
+          "ipython",
+          "matplotlib",
+          "networkx",
+          "numpy",
+          "scikit-image",
+          "scikit-learn",
+          "scipy",
+          "xarray",
+        ]
```

Comment:
These are the project that currently have wheels on https://anaconda.org/scientific-python-nightly-wheels.

Reply:
I feel there should be a master list of the packages separately, rather than hard-wired in here (easier not missing to update when a new one is added)

Reply:
@bsipocz If you do this then you will have to run sequentially instead of in parallel as I don't think(?) that it is possible to set GHA matrix values to variables. If I'm wrong that would be great. (Also feels a bit like once you need config files for a GHA workflow then this should live in a new repository).

Reply:
You make a good point though that ideally we don't need to keep track of anything though and we just query the index itself for the project names.

Reply:
Doing it with a query is great, but anything that is not hard-wired values in the workflow yaml is really an improvement.

Reply:
Agreed, I don't think it's a concern to run this sequentially too. We can easily get the list with something like `anaconda show scientific-python-nightly-wheels`

Reply:
Actually that was a simple over lunch fix, so good suggestion and yeah as @bsipocz pointed out this is actually faster (which in retrospect, if it wasn't there would be problems!).